images/build/Dockerfile

```diff
@@ -158,7 +158,7 @@ RUN set -ex \
  && ln -s 10.10.0 /opt/nodejs/10.10 \
  && ln -s 10.14.2 /opt/nodejs/10.14 \
  && ln -s 10.16.3 /opt/nodejs/10.16 \
- && ln -s 12.9.1 /opt/nodejs/12.9 \
+ && ln -sfn 12.16.1 /opt/nodejs/12.16 \
```

Comment:
what does `fn` mean?

Reply:
create link by force …. I was getting errors while creating lin and the error kept telling me a link already exists. I got rid of the error using this flag

Reply:
create link by force